doc/modules/ROOT/pages/tutorials/graph-analytics-serverless.adoc

```diff
@@ -111,6 +111,7 @@ from datetime import timedelta
 from graphdatascience.session import DbmsConnectionInfo
 
 # Identify the AuraDB instance
+# you can also use DbmsConnectionInfo.from_env() to load credentials from environment variables
```

Comment:
Why not use the tooling here and instead call out the required environment variables in prose or as a listing

Reply:
we were debating with @RafalSkolasinski .

Its more explicit if you type out the parameters in the example.

I was also considering a mix of both worlds

```
try:
   DbmsConnectionInfo.from_env()
except Exception:
     DbmsConnectionInfo(...)
```

arguably thats a bit harder to read but might just work out of the box if you set devenv. 
and you can also easily fallback if you dont use env.